onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.py

```diff
@@ -60,3 +60,86 @@
 
 # pylint: disable=wrong-import-position, disable=unused-import, disable=wildcard-import
 from _kernel_explorer import *  # noqa
+
+"""Benchmark Reporter"""
+
+from abc import abstractmethod
+from contextlib import contextmanager
+from dataclasses import dataclass
+
+
+@dataclass
+class InstanceStatus:
+    name: str
+    dtype: str
+    milliseconds_duration: float
+
+    def __lt__(self, other):
+        if "Tunable" in self.name or other.duration < 0:
+            return True
```

Comment:
> other.duration < 0:

What does `duration < 0` signify here?

Reply:
Some test cases will encounter instances that do not support their parameters. we need a flag(e.g. `duration = -1`) to identify which instances do not support, and put these instances lower in the sort.